lessons/9_more_about_functions/index.html

```diff
@@ -0,0 +1,270 @@
+<html>
+
+<head>
+	<title>Lesson 9</title>
+	<link rel="stylesheet" href="../../reveal.js/css/reveal.css">
+	<link rel="stylesheet" href="../../reveal.js/css/theme/moon.css">
+	<link rel="stylesheet" href="../../reveal.js/lib/css/zenburn.css">
+	<link rel="JavaScript" href="../../reveal.js/plugin/markdown/marked.js">
+	<link rel="JavaScript" href="../../reveal.js/plugin/markdown/markdown.js">
+	<style>
+		b {
+			background-color: #ad0170;
+		}
+
+		.reveal pre {
+			margin: 0;
+		}
+
+		.monospace {
+			font-family: monospace !important;
+		}
+
+		.illustration {
+			border: 0px
+		}
+
+		.reveal li {
+			margin-bottom: 0.5em;
+		}
+
+		.reveal pre {
+			width: 100%;
+		}
+
+		.reveal pre code {
+			cursor: pointer;
+		}
+
+		.reveal li pre {
+			width: fit-content;
+		}
+
+		.logo {
+			position: fixed;
+			/* Sit on top of the page content */
+			z-index: 2;
+			/* Specify a stack order in case you're using a different order for other elements */
+		}
+	</style>
+</head>
+
+<body>
+	<a href="https://www.redi-school.org/"><img src="../../redi-school-logo.jpg" class="logo" /></a>
+	<div class="reveal">
+		<div class="slides">
+			<section>
+				<h1>Lecture</h1>
+				<h3>More about function</h3>
+			</section>
+			<section>
+				<h3>What we would cover?</h3>
+				<ul>
+					<li class="fragment">Functions with arguments</li>
+					<li class="fragment">Return statement</li>
+					<li class="fragment">Callback?</li>
+					<li class="fragment">Practice</li>
+				</ul>
+			</section>
+			<section>
+				<h4>So far we wrote functions in the following way</h4>
+				<pre>
+					<code>
+function buttonClickHandler() {
+  const btnEl = document.getElebentById('importantBtn');
+  btnEl.innerHtml = 'No it is not so important';
+}
+					</code>
+				</pre>
+				<p>This function is good but we can't reuse it<p>
+			</section>
+			<section>
+				<h3>Let's look at this example</h3>
+				<p>We want to calculate the distance based on a <b>speed</b> and <b>time</b></p>
+				<pre class="fragment">
+					<code>
+function calculateDistance() {
+	let speed = 60;
+	let time = 2;
+	alert("Distance is: " + speed * time);
+}
+calculateDistance();
+					</code>
+				</pre>
+				<ul class="fragment">
+					<li>Speed and time always the same</li>
+					<li>We can only alert the distance</li>
+				</ul>
+			</section>
+			<section>
+				<h3>But we can improve!</h3>
+			</section>
+			<section>
+				<h3>Function arguments</h3>
+				<p class="fragment"><b>Arguments</b> are like variable boxes which you can use inside function</p>
```

Comment:
```suggestion
				<p class="fragment"><b>Parameters</b> are like variable boxes which you can use inside function</p>
```
There is a difference between arguments and parameters. 
A parameter is a variable in a function definition
An argument  is the data you pass into the parameters when you call the function

Reply:
True! good catch